project/core/templates/how_it_works.html

```diff
@@ -4,7 +4,7 @@
 
 <!-- How it Works -->
 {% block extra_css %}
-<link type="text/css" rel="stylesheet/less" href="{% static " less/how_it_works.less" %}" />
+<link type="text/css" rel="stylesheet/less" href="{% static "less/how_it_works.less" %}" />
```

Comment:
the space had to be removed here and in about.html otherwise I would get a file error saying how_it_works.less is not found.